### ISSUE TITLE
Should SA1118 be reported for for verbatim strings?

### ISSUE THREAD

**Issue by NikolayIT** [DotNetAnalyzers/StyleCopAnalyzers#1881]
Should SA1118 be reported here? (currently it is reported by StyleCopAnalyzers)

```cs
[InlineData(
@"[Foo][Bar, Car]",
@"[Foo]
[Bar]
[Car]")]
```

**Comment by sharwell** [comment]
I've never really thought about it. What does StyleCop Classic do?

**Comment by NikolayIT** [comment]
The classic StyleCop does not report it

**Comment by vweijsters** [comment]
I don't think that it should be reported, as there is no easy fix. Verbatim string literals can only be replaced by regular string literals (due to limitations on attributes) and that would seriously require the user to reformat the string, thereby (potentially) forcing him/her to reduce readability due to the additional `\r\n` characters needed.

**Comment by sharwell** [comment]
Marking as wontfix based on the following:

1. It matches StyleCop Classic
2. I don't see an obvious benefit to deviating

I'll reopen if more people start to move in the other direction.

**Comment by NikolayIT** [comment]
Just wanted to mention that as I wrote in https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/1881#issuecomment-161084336 StyleCop classic doesn't report any problems in attributes at all.
